packages/scss/src/components/_header.scss

```diff
@@ -9,3 +9,12 @@
 	position: relative;
 	z-index: 1;
 }
+
+.header-left {
```

Comment:
It should be `header-content-left/right` otherwise we might think it is a header that lives on the left/right side